accounts/static/accounts/js/profile.js

```diff
@@ -0,0 +1,392 @@
+// Function taken from landing_page.js
+async function getCSRFToken() {
+  try {
+    const response = await fetch("/get-csrf-token/");
+    const data = await response.json();
+    const csrfToken = data.csrf_token;
+    return csrfToken;
+  } catch (error) {
+    console.error("Error fetching CSRF token:", error);
+    throw error; // Rethrow the error to propagate it
+  }
+}
+// Goes to movie page when user clicks movie poster
+function goToMoviePage(movieId) {
+  window.location.href = "/movie/${movieId}";
+}
+
+// Overlay element for background blur
+const overlay = document.querySelector(".overlay");
+
+/****************************************************
+ *
+ * Remove friend functionality
+ *
+ ***************************************************/
+
+const remove_friend_modal = document.getElementById("remove_friend_modal");
+const remove_friend_name = document.getElementById("remove_friend_name");
+const remove_friend_button = document.getElementById("remove_confirm_button");
+const remove_reject_button = document.getElementById("remove_reject_button");
+const remove_friend_response = document.getElementById(
+  "remove_friend_response"
+);
+
+/**
+ * This function opens/resets the modal for removing a friend
+ * from the users friends list.
+ * @param {String} username the username of the friend that may be removed
+ */
+function openRemoveFriendModal(username) {
+  // Make accept/reject buttons visible
+  remove_friend_button.style.display = "inline-block";
+  remove_reject_button.style.display = "inline-block";
+
+  // Update text with friend's username
+  remove_friend_name.innerHTML =
+    "Would you like to remove " + username + " from your friends list?";
+
+  // Make modal visible, blur background
+  overlay.classList.remove("hidden");
+  remove_friend_modal.style.display = "block";
+
+  // Add proper functionality to button
+  remove_friend_button.onclick = function () {
+    removeFriend(username);
+  };
+}
+
+/**
+ * This function closes the modal for removing a friend from the users friends list.
+ */
+function closeRemoveFriendModal() {
+  // Make modal invisible
```

Comment:
comments right after function declarations seem... pointless when you have a header you know?
like "this function makes the modal invisible for this" or whatever instead - sum up both in the header